src/frontend/src/redux/records/reducer.ts

```diff
@@ -1,22 +1,20 @@
-import { RecordsState, LOAD_RECORDS, RecordActionTypes } from './types';
+import {
+  LOAD_SEARCH_RECORDS,
+  SearchRecordState,
+  SearchRecordsActionType
+} from './types';
 
-const initialState: RecordsState = {
+const initalState: SearchRecordState = {
   records: []
 };
 
 export function recordsReducer(
-  state = initialState,
-  action: RecordActionTypes
-): RecordsState {
+  state = initalState,
+  action: SearchRecordsActionType
+): SearchRecordState {
   switch (action.type) {
-    case LOAD_RECORDS:
-      // The new state is the records returned in the
-      // action. We ignore existing records and do not
-      // necessarily include them in the new state. This
-      // is a "destructive update".
```

Comment:
Why'd you get rid of this comment?

Reply:
Originally I made another reducer.ts file since I wasn't sure if the original file was intended for what I was doing, and I didn't want to alter it.  I then replaced the original file with what we currently have, and overlooked the comments. Comments are back in.